iot/config/connect.rst

```diff
@@ -20,35 +20,42 @@ Follow the steps to connect your IoT Box.
 Ethernet Connection
 ~~~~~~~~~~~~~~~~~~~
 
-1. Connect to the IoT Box all the devices that have to be connected with
-   cables (ethernet, usb devices, etc.).
+* Connect to the IoT Box all the devices that have to be connected with
+  cables (ethernet, usb devices, etc.).
 
-2. Power on the IoT Box.
+* Power on the IoT Box.
 
-3. Then click on the Scan button.
+* Read the Pairing Code from a screen or a receipt printer connected to the IoT Box.
```

Comment:
You can keep the numberings (in WiFi Connection too).
Just indent the `image` directive with 3 spaces so that it aligns with this line and it should work just fine. Also, you can remove the `:align: center` since the image is now part of a list item.

Reply:
I would, too, keep the numbering :)

Reply:
Wouldn't it be strange to remove the `:align: center` as other images on the page will be centered?

Reply:
I think it will be automatically centered because it is part of a list item, am I right? @AntoineVDV

Reply:
No, it won't be automatically centered. But the general guideline for lists' images is to align them to the left because they are usually quite small and the page looks weird if they are centered.